.github/nightly_matrix.php

```diff
@@ -91,6 +91,33 @@ function get_windows_matrix_include(array $branches) {
     return $jobs;
 }
 
+function get_macos_matrix_include(array $branches) {
+    $jobs = [];
+    foreach ($branches as $branch) {
+        foreach([true, false] as $debug) {
+            foreach([true, false] as $zts) {
+                $jobs[] = [
+                    'branch' => $branch,
+                    'debug' => $debug,
+                    'zts' => $zts,
+                    'os' => $branch === 'master' ? '13' : '12',
+                    'arch' => 'X64',
+                ];
+                if($branch['version']['minor'] === 4) {
```

Comment:
```suggestion
                if ($branch['version']['minor'] >= 4 || $branch['version']['major'] >= 9) {
```

Would be nice if we could use `version_compare` but the parts are split because they're otherwise not comparable by the github action expression language.